apps/avifenc.c

```diff
@@ -115,7 +115,7 @@ static void syntax(void)
     printf("    -k,--keyframe INTERVAL            : Set the maximum keyframe interval (any set of INTERVAL consecutive frames will have at least one keyframe). Set to 0 to disable (default).\n");
     printf("    --ignore-exif                     : If the input file contains embedded Exif metadata, ignore it (no-op if absent)\n");
     printf("    --ignore-xmp                      : If the input file contains embedded XMP metadata, ignore it (no-op if absent)\n");
-    printf("    --ignore-icc                      : If the input file contains an embedded ICC profile, ignore it (no-op if absent)\n");
+    printf("    --ignore-profile,--ignore-icc     : If the input file contains embedded color profile info, ignore it (no-op if absent)\n");
```

Comment:
Nit: We can call this "an embedded color profile", i.e., just change "ICC" to "color". The word "info" can be omitted.